examples/join.eve

```diff
@@ -0,0 +1,16 @@
+```
+  commit
+       [#foo name:"a" level:2]
+       [#foo name:"zkp" level:3]
+       [#foo name:"parg" level:0]
+       [#foo name:"naxxo" level:1]
+```
+
+```
+   search [#foo name level]
```

Comment:
We should toe the (unfortunately unwritten) style guide in our public examples. @cmontella (we should write that at some point!). We avoid putting patterns on the same line as actions and always include a newline between the previous action's body and the next action. Otherwise this looks :+1: